Gen2-Sample/CSharp-TSI-Gen2-Sample/DataPlaneClient/azure-rest-api-specs/specification/timeseriesinsights/data-plane/Microsoft.TimeSeriesInsights/stable/2020-07-31/examples/QueryAggregateSeriesWithCategoricalInterpolatedVariable.json

```diff
@@ -0,0 +1,250 @@
+{
+  "parameters": {
+    "api-version": "2020-07-31",
+    "environmentFqdn": "10000000-0000-0000-0000-100000000109.env.timeseries.azure.com",
+    "continuationToken": "aXsic2tpcCI6MTAwMCwidGFrZSI6MTAwMH0=",
+    "parameters": {
+      "aggregateSeries": {
+        "searchSpan": {
+          "from": "2019-10-10T23:41:04.021Z",
+          "to": "2019-10-10T23:42:22.846Z"
+        },
+        "timeSeriesId": [
+          "Sensor_58"
+        ],
+        "interval": "PT2S",
+        "inlineVariables": {
+          "Status_String": {
+            "kind": "categorical",
+            "value": {
+              "tsx": "$event.[Status].String"
```

Comment:
We should update the TSX in each of these samples to use new syntax. 

tsx here would be $event.Status.String or $event['Status'].String